### DISCUSSION TITLE
Monitor flashes off and on, approx every 2-3 minutes.

### DISCUSSION THREAD

**Discussion by WalBal** [MonitorControl/MonitorControl#916] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Monitor flickers off and on twice, approximately every 3 minutes.

System: Macbook Pro M1 (2021, 14 inch)
Port: Display Port via CalDigit T1+ dock
Monitor: Dell S3221QS

Everything works as expected - volume control (which was my main reason for trying MonitorControl), brightness and contrast - all work as expected. However, the monitor turns off (goes black), back to display, back to black and then stays on again. This happens approximately every 3 minutes.

### Expected behavior

Monitor to remain on throughout.

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Monitor Cable(s)/Connection(s):
- Apple Silicon/M1 (yes or no):
```

**Comment by waydabber** [comment]
If this happens only after issuing DDC commands via MonitorControl, then it is probably a dock (or maybe a display) firmware issue (or an unlucky combination of both) I am afraid. You can try using a different app like m1ddc or Lunar and see if that causes a similar misbehavior.

I'll convert this issue into a discussion, we can continue it there.

**Reply by WalBal** [reply]
Many thanks. I've now timed the flickers: They occur every 4min 48sec exactly. No DDC commands issued during this period, and no problem when using m1ddc. Will try Lunar next.

**Reply by waydabber** [reply]
Strange. Btw - if you have the macbook lid open and syncing enabled, that will issue DDC commands as well. You might want to try that.

**Reply by WalBal** [reply]
After an enourmous amount of troubleshooting, I think I have sorted the issue. My Dell monitor has the ability to run at refresh rates up to 60Hz. However, when running it at max, the monitor blinks on and off every second - clearly not an option. I turned it down to 30Hz, and the problem went away. After installing MonitorControl, the monitor settings went back to its default somehow (50Hz), which resulted in the 4min 48sec interval before flashing. I've now reset it to 30Hz, which is fine for the work I do, and everything is stable.

**Reply by waydabber** [reply]
This might be a cable issue as well.

**Reply by WalBal** [reply]
Just tried a new cable (different brand) - same results. Or rather, almost the same. The time between flashes has increased, but the problem remains.

**Reply by waydabber** [reply]
Well then this seems to be a display issue. Strange issue.
